frontend/src/metabase/visualizations/components/skeletons/StaticSkeleton/StaticSkeleton.styled.tsx

```diff
@@ -14,9 +15,15 @@ export const SkeletonTitle = styled(Ellipsified)`
   font-weight: bold;
 `;
 
-export const SkeletonDescription = styled(Ellipsified)`
+export const TruncatedMarkdown = styled(Markdown)`
```

Comment:
nit: both truncated markdown components look very similar, it would be nice to reduce duplication if possible and makes sense

Reply:
Extracted `metabase/core/MarkdownPreview` component: f6106ffe4793c35d3e017a4fd6fbe4ce93546b4d + 7bbc68bf716654f19c6b76ba33865e7cdf1b0d69
Requesting re-review as this is the first component that I'm adding.